product_management/customer_requests_workflow.md

```diff
@@ -39,29 +42,54 @@ On the contrary, understanding the real business need might seem trivial, but re
 
 At times the solution is the perfect one, but in general it will be sub-optimal in terms of efficiency and effectiveness.
 
 ### Solutions to the XY problem
+
 Just look for the most upvoted comment, of course:
 
 ![](5whys.png)
 
 The [5 Whys technique](https://en.wikipedia.org/wiki/5_Whys) is typically used to find the root cause of a problem in a chain of cause-effect.
 
-We can follow a similar approach, asking our customer **why** they need a push notification system. And when they tell you "to let users know this happened", ask **why** they need to know. And when you discover that they need to know so that they can react right away, ask **why** they need to react so quickly. After only 3 whys, you might discover that a reaction time of 1 week is perfectly fine. 
+We can follow a similar approach, asking our customer **why** they need a push notification system. And when they tell you "to let users know this happened", ask **why** they need to know. And when you discover that they need to know so that they can react right away, ask **why** they need to react so quickly. After only 3 whys, you might discover that a reaction time of 1 week is perfectly fine.
 
 And you can simply add an email to the existing email notification module. Turns out the customer solution was inefficient.
 
-Or maybe they want to add an additional field in a form, **why**? *Because we need that information.* **Why**? *Because from that information we get relevant metrics.* And you say.. *do you know how many people actually complete those extra fields? 5%*.
+Or maybe they want to add an additional field in a form, **why**? _Because we need that information._ **Why**? _Because from that information we get relevant metrics._ And you say.. _do you know how many people actually complete those extra fields? 5%_.
 
 Ah.
 
 Well, turns out we need to do something much more complex and costly to get that information. In this case, the customer solution was not effective.
 
+### User Stories as a way to communicate requests
+
+It is encouraged to write user stories in Trello to better define a feature. If possible, the customer should be trained to write user stories themselves.
+
+Without going into much detail, a user story is composed of:
+
+* who: **as a **student
+* what: **I want to **download my schedule in PDF
+* why: **so that **I can print it and bring it too school, where we don't have Wi-Fi
+
+This way of expressing requests gives us insight as to why the feature is necessary, and who will be the user of this feature. This lets us optimize our requirements gathering discussion.
+
+**Important: **User stories are useful both for external and internal requests. Many times the line between a customer and us is blurred in PRISMA. Someone in the back-end team might act as a customer w.r.t. the front-end team, of more often the PM acts as the customer. User stories are still suggested as a way to express requests.
```

Comment:
`w.r.t` cosa vuol dire? `with reference to`? come on! scriviamolo per intero! sono 12 lettere in più :D (ci ho messo svariati secondi a capirlo, sempre che lo abbia capito correttamente)